README.md

```diff
@@ -11,10 +11,25 @@ You can use this action to build images from an [Okteto Manifest](https://www.ok
 
 ## Inputs
 
+### `tag`
```

Comment:
The inputs where removed as "not suported" although they exist at the actions.yaml in order to preserve compatibility with older workflows

Reply:
all the inputs I added here are active on the latest version of the Okteto CLI. Can we add them?